server/index.js

```diff
@@ -0,0 +1,23 @@
+
+const Controller = require('./controllers.js')
+const express = require('express')
+const path = require('path')
+const app = express()
+const port = 8080
+var bodyParser = require('body-parser')
+
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(bodyParser.json())
+app.use(express.static(path.join(__dirname, '../client/dist')))
+
+app.get('/reviews', function (req, res) {
+  Controller.get(req, res)
```

Comment:
It seems like you've already refactored controllers into its own module. You could use express.Router() in your controller.js file and then use app.use('/reviews', Controller) to clean this up a little.